parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -82,7 +102,7 @@ $global:SubscriptionId = "{{GetVariable "subscriptionId"}}"
 $global:ResourceGroup = "{{GetVariable "resourceGroup"}}"
 $global:VmType = "{{GetVariable "vmType"}}"
 $global:SubnetName = "{{GetVariable "subnetName"}}"
-$global:MasterSubnet = "{{GetWindowsMasterSubnetARMParam}}"
+$global:MasterSubnet = "{{GetParameter "masterSubnet"}}" 

```

Comment:
Why do we need care master subnet at all ?

Reply:
This is borrowed from aks-engine, our first intention is to make sure the customdata generated is the one from aks-engine

Reply:
The next step of our PR is to use template variables instead of ARM variables to generate CSE command and remove unnecessary variables

Reply:
We don't need mastersubnet, will remove this field next time.

Reply:
why not in this PR, it's a simple change.

Reply:
Because I have to test this PR overall, even though the deletion seems to be no harm. Another reason is this PR has been for months, I want to close this PR as the first phase.

Reply:
Hmmm, we have to keep it for now, I just found it's referenced in https://github.com/Azure/aks-engine/blob/master/staging/provisioning/windows/kubeletstart.ps1#L9

Reply:
mastersubnet is used in signed scripts for the case of kubenet, AKS doesn't support Kubenet, we can remove this parameter. Also per Tony, this parameter is confusing from the naming and further usage, we don't have such master things in AKS.